x/staking/proto/cosmos/staking/v1beta1/staking.proto

```diff
@@ -79,6 +78,9 @@ message Description {
   string security_contact = 4;
   // details define other optional details.
   string details = 5;
+  // metadata is a map of string keys to any values that can be used to store additional information about the
+  // validator.
+  map<string, google.protobuf.Any> metadata = 6;
```

Comment:
Wallet and explorer complained that proposal metadata were too loose and not typed (even though we had recommended json structure).
Shall we instead be opinionated about which metadata can be set? Using an any will lead to the same complaints I think.